symphony/bdk/core/service/presence/presence_service.py

```diff
@@ -0,0 +1,176 @@
+from typing import List
+from enum import Enum, auto
+
+from symphony.bdk.core.auth.auth_session import AuthSession
+from symphony.bdk.gen.pod_api.presence_api import PresenceApi
+from symphony.bdk.gen.pod_model.v2_presence import V2Presence
+from symphony.bdk.gen.pod_model.v2_presence_status import V2PresenceStatus
+from symphony.bdk.gen.pod_model.v2_user_presence import V2UserPresence
+
+
+class PresenceStatus(Enum):
+    """The list of all possible values for the presence status.
+    Set Presence <https://developers.symphony.com/restapi/reference#set-presence>`_
+    """
+
+    def _generate_next_value_(name, start, count, last_values):
```

Comment:
This is fine but you could also use the `value` field to get the enum name as String, like `PresenceStatus.AVAILABLE.value` will return `"AVAILABLE"`.

Reply:
sure, I just thought that using value can be more concise. I will change it and remove the overridden method